src/force/CabanaPD_Force_PMB.hpp

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * Copyright (c) 2022-2023 by Oak Ridge National Laboratory                 *
+ * Copyright (c) 2022 by Oak Ridge National Laboratory                 *
```

Comment:
The star on the right does not look aligned her, but it looks OK in my local copy.